recipe/meta.yaml

```
@@ -45,24 +50,46 @@ outputs:
         - cachetools >=4.0,<6
```

```suggestion
        - cachetools >=4.0,<7
```
https://github.com/streamlit/streamlit/blame/1.46.0/lib/setup.py#L36C6-L36C25